host-controller/src/main/java/org/jboss/as/host/controller/Main.java

```diff
@@ -445,6 +446,12 @@ public static HostControllerEnvironmentWrapper determineEnvironment(String[] arg
                 } else if (arg.equals(CommandLineConstants.SECMGR)) {
                     // Enable the security manager
                     securityManagerEnabled = true;
+                } else if (arg.startsWith(CommandLineConstants.STABILITY)) {
```

Comment:
See my comments in server module Main.

Reply:
Fixed.  Also removed from command line usage, when stability has only 1 possible value.